client/components/Coverages/CoverageEditor/CoverageFormHeader.tsx

```diff
@@ -173,27 +187,28 @@ export class CoverageFormHeaderComponent extends React.PureComponent<IProps> {
                         <ListRow>
                             <Button
                                 text={gettext('Reassign')}
-                                className="btn btn--hollow btn--small"
                                 onClick={this.showAssignmentModal}
-                                tabIndex={0}
-                                enterKeyIsClick
-                                disabled={!!addNewsItemToPlanning}
-                                autoFocus
+                                style="hollow"
+                                size="small"
+                                expand
                             />
                         </ListRow>
-                        {!onRemoveAssignment ? null : (
-                            <ListRow>
-                                <Button
-                                    text={gettext('Remove')}
-                                    className="btn btn--hollow btn--small"
-                                    onClick={onRemoveAssignment}
-                                    tabIndex={0}
-                                    enterKeyIsClick
-                                    disabled={!!addNewsItemToPlanning}
-                                    autoFocus
-                                />
-                            </ListRow>
-                        )}
+                        {onRemoveAssignment != null
+                            && !isAssignmentLocked
```

Comment:
We can probably add this condition for the re-assign as well. If the Assignment is locked, the user should not be able to re-assign it to someone else